common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -81,7 +81,7 @@ public void renderText(
         int line = 0;
         int partBegin = 0;
 
-        for (int i = parts.size() - 1; i >= 0 && partBegin < parts.size() - 1; i--) {
```

Comment:
Maybe this is a simple as it can get, and/or maybe it doesn't matter, but this is really complicated logic. As you might have noticed, I have this "don't make me think!" attitude about code. :-) Everything that is not immediately clear to me is "sus". :-D

I wonder if it's possible to make the logic clearer. For one, are you rendering this "backwards"? Why?

(I did not really react on this when you created the code, but now when you change it I realize I don't understand what your fix is doing...)

Reply:
If you say "whatever, not important, leave it be" I will accept that, too. I know we have more important issues to address.

Reply:
The loop is backwards so we can get the biggest part that we can fit in the maxWidth.

Reply:
I think this loop always has to be a bigger one, I will add a clarifying comment for the best of the two worlds.